src/streams/UnionStream.js

```diff
@@ -0,0 +1,29 @@
+import { Writable } from 'stream';
+
+class UnionStream extends Writable {
```

Comment:
I'm going to try rewriting this as not a stream tomorrow because after playing around with extending streams for the `loggly stream` I don't really see the benefit of having a stream here because the union stream (as used) is a union of `process.stdout` which is a stream and has all the functions stream has and the loggly client which I think does it's own buffering internally anyways. I will clean up naming tomorrow.